src/components/Header.tsx

```diff
@@ -1,30 +1,34 @@
 import React from 'react';
+import Container from 'react-bootstrap/esm/Container';
+import Nav from 'react-bootstrap/esm/Nav';
+import Navbar from 'react-bootstrap/esm/Navbar';
+import { NavLink } from 'react-router-dom';
 
 function Header() {
   return (
-    <nav className="navbar navbar-light">
-      <div className="container">
-        <a className="navbar-brand" href="index.html">conduit</a>
-        <ul className="nav navbar-nav pull-xs-right">
-          <li className="nav-item">
-            <a className="nav-link active" href="">Home</a>
-          </li>
-          <li className="nav-item">
-            <a className="nav-link" href="">
-              <i className="ion-compose"></i>&nbsp;New Post
-            </a>
-          </li>
-          <li className="nav-item">
-            <a className="nav-link" href="">
-              <i className="ion-gear-a"></i>&nbsp;Settings
-            </a>
-          </li>
-          <li className="nav-item">
-            <a className="nav-link" href="">Sign up</a>
-          </li>
-        </ul>
-      </div>
-  </nav>
+      <Navbar >
+        <Container>
+          <Navbar.Brand href="/#/">conduit</Navbar.Brand>
+          <Nav className="nav navbar-nav pull-xs-right">
+            <Nav.Item className="nav-item">
+              <NavLink className="nav-link" to="/#/">Home</NavLink>
+            </Nav.Item>
+            <Nav.Item className="nav-item">
+              <NavLink className="nav-link" to="/#/article">
```

Comment:
If you are wanting to have the `#` (hash) in the URL I would recommend using the HashRouter instead since it will automatically add that. If you do not want the url hash I would recommend removing it from here and the router.

Reply:
Used the hash router and removed the hash literals from the routes